### PR TITLE
fix(dashboards): Don't reload data when updating widgets

### PR DESCRIPTION
**Pull request by shruthilayaj**
Dashboard page was being reloaded on save
but doesn't need to be. The PUT request can
be made async and in the background and the
UI update can happen right away because we
already know the state.

### CODE REVIEW

**Review by narsaynorath — COMMENTED** [review]
Just some small comments but I love that it's not reloading each time now!

File: static/app/views/dashboardsV2/view.tsx [hunk 1]
```diff
@@ -66,6 +66,7 @@ function ViewEditDashboard(props: Props) {
                 }
                 return reloadData(...args);
```
[the remark this answers is not in the document]
Comment (shruthilayaj) [reply static/app/views/dashboardsV2/view.tsx:67]: yeah I think it's good to remove, I was planning to do it as a follow up once i make sure this change works okay in production!

[the remark this answers is not in the document]
Comment (narsaynorath) [reply static/app/views/dashboardsV2/view.tsx:67]: Sounds good to me!

**Review by narsaynorath — APPROVED** [review]
+1, looks good to me! the test updates are great.

**Pull request merged**